firestore/schedule-export/index.js

```diff
@@ -0,0 +1,28 @@
+const firestore = require('@google-cloud/firestore');
+const client = new firestore.v1.FirestoreAdminClient();
+const bucket = process.env.FIRESTORE_EXPORT_BUCKET;
+
+exports.scheduledFirestoreExport = (event, context) => {
+  const databaseName = client.databasePath(
+    process.env.GCLOUD_PROJECT,
```

Comment:
Can we use `GCP_PROJECT` because `GCLOUD_PROJECT` is deprecated, see https://cloud.google.com/functions/docs/env-var